### DISCUSSION TITLE
Can I build iLogtail on my MacBookPro?

### DISCUSSION THREAD

**Discussion by guhanjie** [alibaba/ilogtail#275] [Help] [answered]
Can I build iLogtail on my MacBookPro?

**Comment by yyuuttaaoo** [comment, marked as answer]
iLogtail consists of core and plugin. The plugin part is written in Go, so you can build it on Mac. The core part is written in C++ and requires a build image to build. The ilogtail-build image has not been tested on MacBookPro. If you are using an Intel Mac, theoretically it should work. It doesn't support M1 Mac yet.
Arm arch support is on the roadmap of iLogtail. So you are welcome to join us if you want to accelerate the development.
